src/constants.ts

```diff
@@ -43,24 +44,54 @@ export const REACTOR_ADDRESS_MAPPING: ReactorMapping = {
   1: {
     [OrderType.Dutch]: "0x6000da47483062A0D734Ba3dc7576Ce6A0B645C4",
     [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
+    [OrderType.Relay]: "0x0000000000000000000000000000000000000000",
   },
   5: {
     [OrderType.Dutch]: "0x6000da47483062A0D734Ba3dc7576Ce6A0B645C4",
     [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
+    [OrderType.Relay]: "0x0000000000000000000000000000000000000000",
   },
   137: {
     [OrderType.Dutch]: "0x6000da47483062A0D734Ba3dc7576Ce6A0B645C4",
     [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
+    [OrderType.Relay]: "0x0000000000000000000000000000000000000000",
   },
   12341234: {
     [OrderType.Dutch]: "0xbD7F9D0239f81C94b728d827a87b9864972661eC",
     [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
+    [OrderType.Relay]: "0x0000000000000000000000000000000000000000",
+  },
+};
+
+export const REACTOR_CONTRACT_MAPPING: ReactorMapping = {
```

Comment:
can we add like a `sameAddressMApping` thing since gouda is same on most chains?

Reply:
and relayer likely will be too

Reply:
yes! can do in a follow up